libs/core-ui/src/lib/util/getFeatureOptions.ts

```diff
@@ -14,14 +14,17 @@ export function getFeatureOptions(
       const key = JointDataset.DataLabelRoot + index.toString();
       const meta = jointDataset.metaDict[key];
       const options = meta.isCategorical
-        ? meta.sortedCategoricalValues?.map(
-            (optionText: string | number, index: number) => {
+        ? meta.sortedCategoricalValues
+            ?.filter(
+              (optionText: string | number, _: number) =>
+                optionText !== undefined && optionText !== null
+            )
+            .map((optionText: string | number, index: number) => {
               if (typeof optionText !== "string") {
                 optionText = optionText.toString();
```

Comment:
If it's a missing value, like defined in the example data, which is what the customer had in their dataset.  I assume it could be null or undefined, perhaps it's always only null in reality, but handling undefined just in case seems like a good idea.